### ISSUE TITLE
Wiki

### ISSUE THREAD

**Issue by TobiasKletter** [tischi/spindle3d#24]
Ideas for our wiki

Quick start manual
- Installing the plugin
 - How to analyse a single image
 - How to batch analyse a whole dataset
 - What's in the output image and table

Advanced manual

- What parameters can be adjusted in the advanced version
- Requirements for cell volume measurements

Trouble shooting tips & tricks
- Limits of the analysis (when is the image quality simply not good enough...)
- Advice on spindle cropping
- Advice on quality control 

Methods
- Basically the description of the workflow from the manuscript
- I can provide some additional drawings / schematics for key steps

@tischi 
Anything important missing? Is it ok if I go ahead and draft it like this? I would start with the bits and pieces that are already here on the Github site...

**Comment by tischi** [comment]
Awesome!
I will be happy to assist improving the draft once it is there!

**Comment by TobiasKletter** [comment]
https://docs.google.com/document/d/1unAeSLxTzn8hJkKFiAdKmccHr8hNZ1EEcysxok3Nk8E/edit

**Comment by tischi** [comment]
@TobiasKletter 
Can you ping me once the Wiki GDoc is in a state that I can convert it to the README?
I will do this only once (from them on we should directly modify the README).
In fact you should have to rights to do so.

**Comment by tischi** [comment]
I invited you to the project; you can them directly modify the README online.
But I can do the first version.

**Comment by tischi** [comment]
@TobiasKletter you could also have a look at this for inspiration: https://github.com/bigdataprocessor/bigdataprocessor2

**Comment by TobiasKletter** [comment]
@tischi 
From my side, the Google Doc is good to go!

**Comment by tischi** [comment]
I updated the README.

Feel free to modify yourself (it is in markdown syntax).

For me it currently is a bit strange as it seems to have cashed some of the embedded images and does not show the new versions. Maybe this will look better tomorrow...

https://github.com/atom/markdown-preview/issues/207

**Comment by TobiasKletter** [comment]
Awesome, thanks! Here it looks good already!
